src/Utils/Hooks/useAuthIntent/__tests__/useAuthIntent.jest.ts

```diff
@@ -244,27 +280,96 @@ describe("runAuthIntent", () => {
           objectId: "banksy",
         })
       )
+    })
 
-      jest.spyOn(mediator, "trigger")
+    afterEach(() => {
+      jest.resetAllMocks()
+    })
+
+    describe("success", () => {
+      it("doesn't need a mutation; calls the onSuccess callback and expires the cookie", async () => {
+        jest.spyOn(Cookies, "expire")
+
+        const onSuccess = jest.fn()
+
+        await runAuthIntent({
+          user: {},
+          relayEnvironment: env as any,
+          onSuccess,
+        })
+
+        expect(onSuccess).toBeCalledWith({
+          action: "createAlert",
+          kind: "artist",
+          objectId: "banksy",
+        })
+
+        expect(Cookies.expire).toBeCalledWith("afterSignUpAction")
+      })
+    })
+
+    // There is no failure branch for this action
```

Comment:
TBH, I think we could have allowed the UI to be opened and then had the create button pop open the auth modal with the appropriate intent. I wonder if we should send out toasts for success though.